bgpd/bgp_vty.c

```diff
@@ -2332,6 +2332,38 @@ DEFUN (no_bgp_timers,
 	return CMD_SUCCESS;
 }
 
+/* BGP minimum holdtime.  */
+
+DEFUN(bgp_minimum_holdtime, bgp_minimum_holdtime_cmd,
+      "bgp minimum-holdtime (1-65535)",
```

Comment:
As long as hold timer range is 0 to 65535, the range allowed for bgp minimum-hold time command (this command) should also be 0 to 65535. This is how other vendors like Cisco supports.

Hold timer of 0 has meaning where no keep alives are sent between the neighbors.

Reply:
Thank you for comment.
The difference between holdtime and minimum-holdtime is that holdtime defaults to 180 seconds when set to no, so the only way to disable holdtime is to set it to 0, while minimum-holdtime can be disabled by simply setting it to no. Therefore, even if the range of holdtime is 0-65535, there is no need to set the range of minimum-holdtime to 0-65535 as well.

Reply:
When both the neighbors are configured with 0 hold time, then user needs to do "no minimum-holdtime" in order to session to be established. User can't configure minimum-holdtime in this case as the minimum range starts from 1.
Right ?

Reply:
@srimohans which case do you have in mind? When holdtime is set to 0 and minimum-holdtime set to 0?

Reply:
I am talking about hold time set to 0 on both the neighbors. 

Currently minimum-hold time can't be set to zero. The only way is to disable this CLI via "no" in order to set minimum-holdtime to zero.

Reply:
But it's the same `no bgp minimum-holdtime` versus `bgp minimum-holdtime 0`. Or I'm missing something?

Reply:
I would like if we agree with a common solution here :) If Cisco has 0, then we can follow that, but the question would be also if Cisco's implementation is the correct one.

Reply:
I meant, This is a minor nit and I don't want to hold back this PR for that reason. I prefer to start the range for min-hold time as 0, however I m okay to approve this PR in the current form if you guys want to go ahead.